content/_includes/site/alert.njk

```diff
@@ -1,7 +1,7 @@
 {% set alert = collections.all | findPage('data.info_slug', 'alert') %}
 
 {% if alert and (alert.data.live) %}
-  <aside data-layout="alert" data-ccs-colors="dark">
+  <aside data-layout="alert" data-ccs-colors="dark" role="banner" aria-label={{ alert.data.aria }}>
```

Comment:
Why `role=banner`? I think the Complementary role that comes from the aside is probably fine here?

Reply:
Ah okay, that makes sense.